src/Wallabag/CoreBundle/Resources/views/themes/material/public/css/main.css

```diff
@@ -221,15 +221,11 @@ main ul.row {
     padding: 0px 0.75rem;
 }
 
-.data .card .card-content {
+.data .card .card-body {
     height: 22em;
     overflow: hidden;
 }
 
-img.preview {
```

Comment:
if you remove these lines, preview picture display is broken on article view (picture is too huge)